lib/registries/addon/overview/route.ts

```diff
@@ -76,23 +79,68 @@ export default class Overview extends GuidRoute {
                 ),
                 institution: (institutions as SparseModel[]).map(institution => institution.name as string),
             };
-            const headTags = [...this.metaTags.getHeadTags(metaTagsData)];
+
+            const metaTags: HeadTagDef[] = this.metaTags.getHeadTags(metaTagsData);
+
+            // Google Structured Data
+            let jsonLD: object | undefined;
+            let scriptTag;
+            try {
+                jsonLD = await this.returnStructuredData(id);
+                const jsonString: object | undefined = jsonLD ? jsonLD : undefined;
+                if (jsonString) {
+                    scriptTag = {
+                        type: 'script',
+                        content: jsonString,
+                        attrs: {
+                            type: 'application/ld+json',
+                        },
+                    };
+                } else {
+                    scriptTag = undefined;
+                }
+                if (scriptTag) {
+                    metaTags.push(scriptTag);
+                }
+            } catch (e) {
+                const errorMessage = this.intl.t('general.structured_data.json_ld_retrieval_error');
+                captureException(e, { errorMessage });
+            }
+
             if (provider && provider.assets && provider.assets.favicon) {
-                headTags.push({
+                metaTags.push({
                     type: 'link',
                     attrs: {
                         rel: 'icon',
                         href: provider.assets.favicon,
                     },
                 });
             }
-            this.set('headTags', headTags);
+            this.set('headTags', metaTags);
             this.metaTags.updateHeadTags();
         }
-
         blocker.done();
     }
 
+    async returnStructuredData(guid: string): Promise<any> {
+        const path = `${config.OSF.url}/${guid}/metadata/?format=google-dataset-json-ld`;
```

Comment:
The `google-dataset-json-ld` format will need to be switched on to have this code work. I have previously discussed default values as an option for when it is not. It was decided to not have the frontend serve anything in place of the backend's response.